control-plane/pkg/logging/logger.go

```diff
@@ -24,16 +24,16 @@ import (
 	"knative.dev/pkg/logging"
 )
 
-func Logger(ctx context.Context, action string, object metav1.Object) *zap.Logger {
+func CreateReconcileMethodLogger(ctx context.Context, object metav1.Object) *zap.Logger {
```

Comment:
What if we remove these 2 methods and have only NewLogger that takes ctx, object and action?

Reply:
Also I opted for `Create` more than `New` because new always recalls me constructors...

Reply:
I don't like these names they are not really go-ish, NewReconcileLogger?

Reply:
Or perhaps just remove create and method like  logging.ReconcileLogger?

Reply:
Anyway, we can iterate on nits later ;)
/lgtm

Reply:
New is for a constructor, for me it's a no-go here because this is not a constructor of zap.Logger, it's more a wrapper :smile: